common/models/model-method.js

```diff
@@ -10,4 +11,38 @@ module.exports = function(ModelMethod) {
    * @class ModelMethod
    * @inherits WorkspaceEntity
    */
+
+  ModelMethod._shouldEncodeStaticFlagInName = function() {
+    var version = ModelMethod.app.models.Workspace.loopBackVersion;
+    return encodeStaticFlagInName(version);
```

Comment:
nitpick: I would personally inline `encodeStaticFlagInName` here and remove it.

Feel free to ignore this comment though.